charts/opensearch-operator/templates/opensearch-operator-crds.yaml

```diff
@@ -0,0 +1,5 @@
+{{- if .Values.installCRDs -}}
```

Comment:
what is that file ?

Reply:
Since I wanted to add an if-statement for deploying CRDs, it would get annoying to append and prepend that statement in the CRD files, after having run `cp opensearch-operator/config/crd/bases/opensearch.opster.io_*.yaml charts/opensearch-operator/templates/` each time. 
Instead, I copy the manifests to a `files/` folder and import them using `.Files.Glob`.
It's a standard working procedure explained here: https://helm.sh/docs/chart_template_guide/accessing_files/